CHANGELOG.md

```diff
@@ -6,7 +6,10 @@ description: |-
 ---
 
 # VKCS Provider's changelog
+
 #### v0.7.3 (unreleased)
+- Add ability to search by extra_specs param of vkcs_compute_flavor
```

Comment:
Add ability to filter by "extra_specs" attribute in vkcs_compute_flavor data source.

Reply:
done without quotas